corehq/ex-submodules/casexml/apps/case/templates/case/partials/case_hierarchy.html

```diff
@@ -124,7 +127,17 @@
             </div>
           {% endif %}
         </td>
-
+        {% if request|toggle_enabled:"SUPPORT" %}
+          <td>
+            {% if case.case_id == current_case.case_id %}
+              <em>(current case)</em>
```

Comment:
translate?

Reply:
Whoops, missed this one, sorry about that!
https://github.com/dimagi/commcare-hq/pull/34665